.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- Before creating an issue please make sure you are using the latest version. -->
+
+<!-- If this is a feature request use this portion of the template and delete the bug portion -->
+**Describe the *feature* request**
+
+
+**What is motivation or use case for changing this?**
+
+
+**Please mention other relevant information such as the browser version, Node.js version, Lighthouse version, and Operating System.**
+
+
+<!-- If this is a bug report use this portion of the template and delete the feature request portion -->
+
+**Describe the *bug* issue**
```

Comment:
I like this set of questions! WDYT about adding one about relevant existing issues?

```html
<!-- Have you searched for similar open issues? -->
<!-- If yours is a duplicate, consider commenting with additional info or +1'ing instead. -->
Related issues: <!-- #9000 -->
```
or something